### ISSUE TITLE
auto export artboards after document saved

### ISSUE THREAD

**Issue by yuchuanxi** [mathieudutour/git-sketch-plugin#123]
I love the plugin, but i just use the `export artboards`

**my work flow**:

1. edit sketch
2. save sketch (and auto export artboards)
3. use tower/souretree or terminal to push change

the plugin just a good wife, help me background

thank you

**Comment by mathieudutour** [comment]
yes that's a good idea, not too hard to implement:
- Add a new command in manifest.json:
   ```
    {
      "name": "autoExportOnSave",
      "identifier": "autoExportOnSave",
      "script": "./commands/autoExportOnSave.js",
      "handlers" : {
        "actions": {
          "DocumentSaved": "onRun"
        }
      }
    },
    ```
- Add an option to auto export on save in the preferences
- create `autoExportOnSave.js` which check for the preference and export

Do you want to send a PR for it or shall I do it?

**Comment by yuchuanxi** [comment]
i can do it, but i need to use little unelegant code

**Comment by mathieudutour** [comment]
published in 0.11.1.
 Thanks again!
